### DISCUSSION TITLE
Not getting response after redirecting to the state bot

### DISCUSSION THREAD

**Discussion by Nik720** [Samagra-Development/SamagraX-community#43] [Troubleshooting and Clarifications]
Hi @aayushkedia95 @ChakshuGautam

Facing an issue after redirecting to the state bot, I am not able to proceed further 
If I initiate conversation directly with the State bot then it is working fine (I have already shown an issue to Chakshu in the call)

Here are the bot details

**Bot ID: 06801678-5ce9-470b-b2f0-eee070813774
formID: "global_form"**
form: https://docs.google.com/spreadsheets/d/1ZrK77k6SMMis7jLoqlqxVEvtP6OTKvJCzJhtaUptqNM/edit?usp=sharing

**Bot ID: 563b8fca-7462-4bf3-819c-0111d377681d
formID: "sunbird-karnataka-bot"**
form: https://docs.google.com/spreadsheets/d/184OoCYrVEuv8xAh4O7xWOxi9upqUJ9Dmu5993CfAXMM/edit?usp=sharing

Please have look into below video 
https://drive.google.com/file/d/1cz6F7X1TE305123_HxkuFnakEwACJaST/view?usp=sharing

cc - @vinukumar-vs

**Comment by Nik720** [comment]
@ChakshuGautam Is there any update on this?

cc - @vinukumar-vs

**Reply by aayushkedia95** [reply]
Hi @Nik720, we're working on this issue right now. You would be able to test this tomorrow.

**Reply by ChakshuGautam** [reply]
@Nik720 https://github.com/samagra-comms/transformer/issues/10

**Reply by Nik720** [reply]
@ChakshuGautam @aayushkedia95 I am not getting any response, can you please check? 

<img width="300" alt="Screenshot 2021-07-15 at 12 19 24 PM" src="https://user-images.githubusercontent.com/33539271/125742337-94ad7698-3515-484f-8e66-3a4edea5f004.png">

there is one more doubt like 
Suppose from global odk form I have selected any state and it redirects to default odk form (because that state doesn't have its own workflow) and now in default odk form I want to prepare a URL based on the selected state, how can I do this? 

Can I get hidden field value from global odk form to state odk form?

cc - @vinukumar-vs

**Reply by aayushkedia95** [reply]
Hi @Nik720, the sandbox seems to be down. It will be up by today evening.

**Reply by Nik720** [reply]
Okay @aayushkedia95 

> Suppose from global odk form I have selected any state and it redirects to default odk form (because that state doesn't have its own workflow) and now in default odk form I want to prepare a URL based on the selected state, how can I do this?
> 
> Can I get hidden field value from global odk form to state odk form?

can you please help with this scenario?

**Reply by aayushkedia95** [reply]
@Nik720, we're not handling this in 4.1, can we take this up next week please?

**Reply by ChakshuGautam** [reply]
This has been fixed. Please test. Let me know when are testing. I will monitor logs.

**Reply by ChakshuGautam** [reply]
@Nik720 great queries. Would be great if we can discuss them separately in different discussion or issue board? Since we are planning for 4.2 we should be able to include these user stories.

**Reply by Nik720** [reply]
> This has been fixed. Please test. Let me know when are testing. I will monitor logs.

sure @ChakshuGautam I'll ping you when once I start testing

**Reply by Nik720** [reply]
> Would be great if we can discuss them separately in different discussion or issue board?

I'll create new discussion thread for this issue
